Jenkinsfile

```diff
@@ -26,7 +34,15 @@ elifeLibrary {
             elifeGitMoveToBranch commit, 'master'
         }
 
-        stage 'Push image', {
+        stage 'Push release image', {
+            isNew = sh(script: "git tag | grep v${candidateVersion}", returnStatus: true) != 0
+            if (isNew) {
+                def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
+                image.tag(candidateVersion).push()
+            }
+        }
+
+        stage 'Push unstable image', {
             def image = DockerImage.elifesciences(this, 'sciencebeam', commit)
             image.push()
```

Comment:
can we change the name to `sciencebeam_unstable` here?

Reply:
(also need to change downstream to pick up `sciencebeam_unstable` instead - but maybe it's better to run the texture demo based on the latest release instead)

Reply:
You should be able to change it here and the Docker Hub page should be auto-created

Reply:
However, the `DockerImage` class only works on existing objects so I think it needs some special method to also execute a `docker tag .../project:$commit .../project_unstable:$commit` command